Stripe/STPBankSelectionTableViewCell.m

```diff
@@ -95,6 +96,10 @@ - (void)configureWithBank:(STPFPXBankBrand)bankBrand theme:(STPTheme *)theme sel
     // Title label
     self.titleLabel.font = theme.font;
     self.titleLabel.text = STPStringFromFPXBankBrand(self.bank);
+    if (offline) {
+        self.titleLabel.text = [self.titleLabel.text stringByAppendingString:@" - "];
```

Comment:
Would it be better to localize with the dash as part of the string, for example `"%s - Offline"`

I'm thinking of RTL translation scenarios.

Reply:
Ah, good point.